tests/unit/challenges/test_aws_utils.py

```diff
@@ -0,0 +1,199 @@
+import boto3
+import mock
+import os
+
+from moto import mock_ecs
+from datetime import timedelta
```

Comment:
@nikochiko same here for imports they should be lexicographically ordered based on package name

Reply:
Yes, I just copy-pasted that from the reference PR, so I didn't pay much attention. I will fix it right away.